workspace/php2/src/XSolveSecurityBundle/Controller/UserController.php

```diff
@@ -19,17 +20,70 @@ public function RestLoginAction(Request $request) {
       $autorizationManager = $this->get('x_solve_security.xsolve_autorization_manager');
       $authenticatedToken = $autorizationManager->getToken($username, $password);
 
-      if ($authenticatedToken) {
-	 $this->getUser()->assignNewToken();
-	 $this->getDoctrine()->getManager()->flush();
+      if (!$authenticatedToken) {
+
+	 throw new AccessDeniedException('unable to find token');

```

Comment:
Rzucasz exception ``AccessDeniedException`` natomiast wiadomośc w nim sugeruje że to jednak powinien być exception typu ``NotFoundException``, może musisz zmienić typ rzucanego exceptiona, a może musisz zmienić message w Exception, ewentualnie możesz jeszcze wytłumaczyć mi i przekonać mnie że tak ja jest jest dobrze.